samples/net/cloud/tagoio_http_post/src/sockets.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define TAGOIO_SOCKET_MAX_BUF_LEN 1280
+
+struct tagoio_context_t {
```

Comment:
It's not typdef'ed, so there's no need for _t suffix, `struct tagoio_context` should be enough.

Reply:
Reworked!